### PR TITLE
(pencil) Fix update script and installation

### PR DESCRIPTION
**Pull request by Windos**
## Description

This PR adjusts the pencil update script so that it works correctly, and also updates the chocolateyInstall.ps1 script and VERIFICATION.txt to reflect these changes.

This PR also expands a number of aliases in the pencil update script.

## Motivation and Context

At the moment the update for this package fails, due to the assumption that there is a separate 32-bit and 64-bit installers for Windows. This is no longer the case and there is only a single installer for Windows now.

This failure has been noted in #2187.

## How Has this Been Tested?

On a vanilla Windows 10 (fully patched) test VM I verified that running the pencil packages' update.ps1 script fails with the same error as seen when run via this [repository's automation](https://gist.github.com/choco-bot/a14b1e5bfaf70839b338eb1ab7f8226f#pencil).

I retested after making these changes and verified that the update.ps1 script succeeded.

The resulting nupkg successfully installed the pencil application on my test VM, and also in my local copy of Chocolatey test environment.

## Screenshot (if appropriate, usually isn't needed):

N/A

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [X] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [X] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [X] The added/modified package passed install/uninstall in the chocolatey test environment.
- [X] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/pencil/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -5,8 +5,7 @@ $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
 $packageArgs = @{
   packageName    = $Env:ChocolateyPackageName
   fileType       = 'exe'
-  file           = Get-Item $toolsPath\*-i386.exe
-  file64         = Get-Item $toolsPath\*_64.exe
+  file           = Get-Item $toolsPath\Pencil-*.exe
```
Comment (pauby) [inline automatic/pencil/tools/chocolateyInstall.ps1:8]: We avoid doing this now and all for the file name to be explicitly used.

**Review by pauby — CHANGES_REQUESTED** [review]
Thanks for picking this up.

File: automatic/pencil/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (Windos) [reply automatic/pencil/tools/chocolateyInstall.ps1]: I've fixed that up

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
@Windos awesome work! Thanks for your contribution!

**Pull request merged**